keycloak/keycloak-admin-ui
# Admin search

## Motivation
Add Search for admin events as per https://marvelapp.com/prototype/7ca9dhf/screen/73036555 with field names as per old admin console.

![Screenshot 2021-09-01 at 16 46 34](https://user-images.githubusercontent.com/4890675/131702675-761d0064-a394-425f-954c-956a1e281315.png)

Added cypress tests which are passing locally as below: 

<img width="537" alt="Screenshot 2021-09-01 at 15 59 29" src="https://user-images.githubusercontent.com/4890675/131701049-ca00a1a2-0735-4ace-a7e4-8371680d2b30.png">

## Verification Steps

1. Go to /event configs page and enable admin events.
2. Go to /events page, admin tab and open search form, add search criteria e.g set operation types to "UPDATE" and select 'Search admin events' button".
3. Verify that correct search results are displaying and the correct chips are displaying.
4. Remove the chip and reopen the search form and select non-existing operation type e.g "CREATE".
5. Verify the correct search results are displaying (no results should display/empty table) and that the correct chip is displaying.
6. Select the 'refresh' button and verify that all events display.

## Checklist:
